ca/ca_test.go

```diff
@@ -266,7 +267,7 @@ func TestFailNoSerial(t *testing.T) {
 	_, err := NewCertificateAuthorityImpl(
 		testCtx.caConfig,
 		testCtx.fc,
-		testCtx.stats,
+		metrics.NewStatsdScope(testCtx.stats, "CA"),
```

Comment:
I think `testCtx.stats` is now unused. Can you replace it with a scope, and reduce the duplication introduced with the `NewStatsdScope`s?